src/sentry/integrations/jira/integration.py

```diff
@@ -65,23 +65,35 @@
 
 
 class JiraIntegration(Integration, IssueSyncMixin):
-    def get_project_config(self):
+    def get_organization_config(self):
         configuration = [
             {
-                'name': 'resolve_status',
-                'type': 'choice',
-                'allowEmpty': True,
-                'label': _('Jira Resolved Status'),
-                'placeholder': _('Select a Status'),
-                'help': _('Declares what the linked Jira ticket workflow status should be transitioned to when the Sentry issue is resolved.'),
+                'name': 'sync_status_reverse',
+                'type': 'boolean',
+                'label': _('Sync Status from Jira to Sentry'),
+                'help': _("When a Jira ticket is moved to a done category, it's linked Sentry issue will be resolved. When a Jira ticket is moved out of a Done category, it's linked sentry issue will be unresolved."),
             },
             {
-                'name': 'unresolve_status',
-                'type': 'choice',
-                'allowEmpty': True,
-                'label': _('Jira Un-Resolved Status'),
-                'placeholder': _('Select a Status'),
+                'name': 'sync_status_forward',
+                'type': 'super_multichoice',
+                'label': _('Sync Status from Sentry to Jira'),
                 'help': _('Declares what the linked Jira ticket workflow status should be transitioned to when the Sentry issue is unresolved.'),
+                'addButtonText': _('Add Project'),
+                'addDropdown': {
+                    'emptyMessage': _('All projects configured'),
+                    'noResultsMessage': _('Could not find Jira project'),
+                    'items': [],  # Populated with projects
+                },
+                'mappedColumnLabel': _('Jira Project'),
+                'mappedKeys': ('on_resolve', 'on_unresolve'),
+                'columnLabels': {
+                    'on_resolve': _('When resolved'),
+                    'on_unresolve': _('When unresolved'),
+                },
+                'multiChoices': {
+                    'on_resolve': {'choices': [], 'placeholder': 'Select status'},
+                    'on_unresolve': {'choices': [], 'placeholder': 'Select status'},
+                },
             },
             {
                 'name': 'resolve_when',
```

Comment:
you can remove `resolve_when` and `unresolve_when` right?

Reply:
Yeah, I'm actually gonna add these as a seperate PR, still a bit of a WIP. Cleaning up some edge states like when it's disabled because it can't talk to Jira

![image](https://user-images.githubusercontent.com/1421724/43099329-b34d0632-8e76-11e8-840a-bb22522458dc.png)